docs/validators/advanced-setups.mdx

```diff
@@ -0,0 +1,102 @@
+title: Advanced Setups
+icon: plug
+---
+
+## Using Remote Signers <Icon icon="key-skeleton-left-right" />
+
+Remote signers can be used with Connect, however only certain versions are compatible.
+
+### Horcrux
+
+#### Required Version
+
+v3.3.0+
+
+#### With `Horcrux-Proxy`
+
+If you are using horcrux-proxy be sure to use v1.0.0+ so that the maxReadSize can be configured in accordance with larger vote-extension payloads
+
+### TMKMS
+
+#### Required Version
+v0.13.1+
+
+
+## Using Distributed Validators <Icon icon="network-wired" />
+
+Connect can be used within a distributed validator setup. To do so, simply apply the same `app.toml` changes to _each_ validator node.
+Head over to [configuration](configuration#application) to see the necessary application-side configurations.
+
+
+## dYdX
+
+Validators for the dYdX chain will need some additional configurations for their sidecar.
+
+### Required Connect Version
+v1.0.1+
+
+### Connect Flags
+In addition to the setup in the [quickstart](quickstart), the following flags are required for dYdX validators running the sidecar:
+
+```shell
+slinky --marketmap-provider dydx_api --market-map-endpoint https://<dydx-node REST API endpoint> --oracle-config oracle.json
+```
+
+### API Keys
+The Raydium provider utilized by dYdX requires API keys to retrieve price data. If you have not received API keys, please reach out in the relevant communication channels.
+
+Below is the necessary file to pass to the `--oracle-config` flag.
+```json oracle.json
+{
+   "providers": {
+      "raydium_api": {
+         "api": {
+            "endpoints": [
+               {
+                  "url": "https://solana.polkachu.com",
+                  "authentication": {
+                     "apiKeyHeader": "x-api-key",
+                     "apiKey": "API KEY"
+                  }
+               },
+               {
+                  "url": "https://slinky-solana.kingnodes.com",
+                  "authentication": {
+                     "apiKeyHeader": "x-api-key",
+                     "apiKey": "API KEY"
+                  }
+               },
+               {
+                  "url": "https://solana.lavenderfive.com",
+                  "authentication": {
+                     "apiKeyHeader": "x-api-key",
+                     "apiKey": "API KEY"
+                  }
+               },
+               {
+                  "url": "https://solana-rpc.rhino-apis.com",
+                  "authentication": {
+                     "apiKeyHeader": "x-api-key",
+                     "apiKey": "API KEY"
+                  }
+               },
+               {
+                  "url": "https://dydx.helius-rpc.com",
+                  "authentication": {
+                     "apiKeyHeader": "x-api-key",
+                     "apiKey": "API KEY"
+                  }
+               }
+            ]
+         }
+      }
+   }
+}
+```
+
+## Neutron
+
+### Required Connect Version
+v1.0.3+
```

Comment:
Says 1.0.1+ above?

Reply:
1.0.1+ is under the dydx heading, this is for neutron